packages/core/src/plugins/socketIO.ts

```diff
@@ -43,8 +44,11 @@ function attachSocketIO(this: Microfleet, opts: any = {}) {
   }
 
   this.socketIO = socketIO
-}
 
+  return {
+    getRequestCount: () => RequestTracker.getRequestCount(service, ActionTransport.socketIO),
```

Comment:
```suggestion
    getRequestCount: RequestTracker.getRequestCount.bind(undefined, service, ActionTransport.socketIO),
```

bind оптимизировали и сейчас быстрее работает чем closure